charts/Controls.tsx

```diff
@@ -623,6 +624,14 @@ export class Controls {
         )
     }
 
+    @computed get hasRelatedQuestions(): boolean {
+        const { chart } = this.props
+        return (
+            !!chart.props.relatedQuestions[0].text &&
+            !!chart.props.relatedQuestions[0].url
```

Comment:
Can we always assume that `relatedQuestions` is non-empty, e.g. `relatedQuestions[0]` is not undefined?
For old charts, too?

Reply:
I think we could as it was getting a default initialisation. Either way, I removed that bit so I'm checking for existence / definition now.